tokens/blocket.se/deprecated.yml

```diff
@@ -70,6 +70,11 @@ color:
     notification:
       text: s-color-text-notification
       background: s-color-background-notification
+    breadcrumbs:
```

Comment:
Regarding this deprecated.yml, is there any use case where we'll deprecate something only for a single brand? if not, it would be easier to have a single deprecated file? WDYT?

Reply:
Yes it would... but as it is now all yaml files are duplicated per brand... so then we would have to change the logic of how the themes are created. 😄

Reply:
Right 👍